app/services/create_proposed_organisation_edit.rb

```diff
@@ -35,7 +35,8 @@ def merge_in_non_published_fields
   end
 
   def send_email_to_superadmin_about_org_edit
-    superadmin_emails = user_klass.superadmins.pluck(:email)
-    mailer_klass.edit_org_waiting_for_approval(organisation, superadmin_emails).deliver_now
+    superadmin_emails = user_klass.superadmin_emails
```

Comment:
@tansaku, line 38 code makes "create_proposed_organisation_edit_spec.rb" failed 4 times when editor is NOT siteadmin. I need some pointers here.

> CreateProposedOrganisationEdit
>   editor is siteadmin
>     creates a proposed organisation edit
>   editor is not siteadmin
>     creates a proposed organisation edit (FAILED - 1)
>     merges in non published fields (FAILED - 2)
>     sends email to superadmin about org edit (FAILED - 3)
>     sets the flash notice on the listener (FAILED - 4)
>